src/Model/Payment/Braintree/CreditCard.php

```diff
@@ -144,7 +141,8 @@ public function manipulateOrderWithPaymentDataBeforeSave(
                              )
                          )->setTokenDetails($this->serializer->serialize($paymentTokenDetails));
 
-            $extensionAttributes->setVaultPaymentToken($paymentToken);
+            $extensionAttributes->setData('vault_payment_token', $paymentToken);
+//            $extensionAttributes->setVaultPaymentToken($paymentToken);
```

Comment:
forgot the comment, although it also complains about setData method ^_^ just not part of the interface I guess. May as well use `setVaultPaymentToken` and just ignore the warning via docblock